@navikt/core/css/baseline/_utilities.css

```diff
@@ -28,6 +28,7 @@
 .navds-sr-only {
   border: 0;
   clip: rect(0, 0, 0, 0);
+  clip-path: inset(50%);
```

Comment:
Hva er bakgrunnen for disse endringene 🤔  (i.e har det vært en bug til nå?)

Reply:
det var en "kjapp" konvertering fra gammelt til nytt (Oppdaget mens Lars og jeg gikk igjennom noe annet), så 'clip' var deprecated.